packages/api-v2/ormconfig.ts

```diff
@@ -18,6 +18,7 @@ const ormconfig: TypeOrmModuleOptions = {
   cli: {
     migrationsDir: "src/migrations",
   },
+  keepConnectionAlive: true,
```

Comment:
removing this caused connection (AlreadyHasActiveConnectionError) errors, I think enabling this option allows for a connection to persist rather than close on app shutdown. maybe it's because there are now multiple app initializations for each test? but since it's not needed in prod, i modified it so that it keys on the env variable for testing